### PR TITLE
Added Frontier and Homestead information to Protocol History

### PR DESCRIPTION
**Pull request by nokcha75**
I added a Frontier section and a Homestead section, including resources for further reading.

## Wiki PR Checklist

Thank you for contributing to the Protocol Wiki! Before you open a PR, make sure to read [information for contributors](https://epf.wiki/#/contributing) and take a look at following checklist:

- [x] Describe your changes, substitute this text with the information
- [x] If you are touching an existing piece of content, ask the original creator for review 
- [x] If you need feedback for your content from wider community, share the PR in our Discord
- [x] Review changes to ensure there are no typos, see instructions bellow

### CODE REVIEW

File: docs/wiki/protocol/history.md [hunk 1]
```diff
@@ -6,9 +6,27 @@ This page highlights important technical changes in the history of Ethereum prot
 
 Useful links: [Overview from Ethereum.org](https://ethereum.org/en/history) and [Meta EIPs from Ethereum.org](https://eips.ethereum.org/meta)
 
+## Frontier
+
+The Frontier release marked the launch of the Ethereum Protocol. It was launched on July 30, 2015, at 3:26:13 AM +UTC, which is the timestamp of the [Ethereum genesis block](https://etherscan.io/block/0). Frontier launched with a gas limit of 5000. This gas limit was hard coded into the protocol to ensure that miners and users could get up and running by installing clients during the initial launch of the protocol. The gas limit would later be increased to 3 million with the Frontier thawing fork (exactly 3,141,592). The canary contracts were contracts that gave a binary signal of either 0 or 1. These canary contracts were an initial launch mechanism used only during the Frontier release of Ethereum. If clients detected that multiple contracts had switched to a signal of 1, they would stop mining and urge the user to update their client. This prevented prolonged outages by ensuring that miners did not prevent a chain upgrade.
+
+Learn more about Frontier in the following resources.
```
Comment (raxhvl) [inline docs/wiki/protocol/history.md:13]:
```suggestion
Learn more about Frontier in the following resources:
```

File: docs/wiki/protocol/history.md [hunk 2]
```diff
@@ -6,9 +6,27 @@ This page highlights important technical changes in the history of Ethereum prot
...
+- https://blog.ethereum.org/2015/08/04/the-thawing-frontier
+- https://web.archive.org/web/20150802035735/https://www.ethereum.org/
+- https://blog.ethereum.org/2015/08/04/ethereum-protocol-update-1
+
 ## Homestead
 
-TODO
+Homestead was the second major release of Ethereum. It introduced [EIP-2](https://eips.ethereum.org/EIPS/eip-2), [EIP-7](https://eips.ethereum.org/EIPS/eip-7), and [EIP-8](https://eips.ethereum.org/EIPS/eip-8). EIP-8 ensures that clients on Ethereum support future network upgrades by introducing devp2p forward compatibility requirements. EIP-7 introduced the DELEGATECALL opcode. EIP-2 contained numerous fixes, such as increasing the gas cost for contract creation via transactions, ensuring that contract creation either succeeded or failed (preventing empty contracts from being created), and modifications to the difficulty adjustment algorithm.
+
+Learn more about Homestead in the following resources.
```
Comment (raxhvl) [inline docs/wiki/protocol/history.md:24]:
```suggestion
Learn more about Homestead in the following resources:
```

File: docs/wiki/protocol/history.md [hunk 3]
```diff
@@ -6,9 +6,27 @@ This page highlights important technical changes in the history of Ethereum prot
...
+
+- https://blog.ethereum.org/2015/07/22/frontier-is-coming-what-to-expect-and-how-to-prepare
+- https://blog.ethereum.org/2015/08/04/the-thawing-frontier
+- https://web.archive.org/web/20150802035735/https://www.ethereum.org/
+- https://blog.ethereum.org/2015/08/04/ethereum-protocol-update-1
+
 ## Homestead
 
-TODO
+Homestead was the second major release of Ethereum. It introduced [EIP-2](https://eips.ethereum.org/EIPS/eip-2), [EIP-7](https://eips.ethereum.org/EIPS/eip-7), and [EIP-8](https://eips.ethereum.org/EIPS/eip-8). EIP-8 ensures that clients on Ethereum support future network upgrades by introducing devp2p forward compatibility requirements. EIP-7 introduced the DELEGATECALL opcode. EIP-2 contained numerous fixes, such as increasing the gas cost for contract creation via transactions, ensuring that contract creation either succeeded or failed (preventing empty contracts from being created), and modifications to the difficulty adjustment algorithm.
```
Comment (raxhvl) [inline docs/wiki/protocol/history.md:22]:
```suggestion
Homestead was the second major release of Ethereum. It introduced [EIP-2](https://eips.ethereum.org/EIPS/eip-2), [EIP-7](https://eips.ethereum.org/EIPS/eip-7), and [EIP-8](https://eips.ethereum.org/EIPS/eip-8). EIP-8 ensures that clients on Ethereum support future network upgrades by introducing devp2p forward compatibility requirements. EIP-7 introduced the `DELEGATECALL` opcode. EIP-2 contained numerous fixes, such as increasing the gas cost for contract creation via transactions, ensuring that contract creation either succeeded or failed (preventing empty contracts from being created), and modifications to the difficulty adjustment algorithm.
```

File: docs/wiki/protocol/history.md [hunk 4]
```diff
@@ -6,9 +6,27 @@ This page highlights important technical changes in the history of Ethereum prot
 
 Useful links: [Overview from Ethereum.org](https://ethereum.org/en/history) and [Meta EIPs from Ethereum.org](https://eips.ethereum.org/meta)
 
+## Frontier
+
+The Frontier release marked the launch of the Ethereum Protocol. It was launched on July 30, 2015, at 3:26:13 AM +UTC, which is the timestamp of the [Ethereum genesis block](https://etherscan.io/block/0). Frontier launched with a gas limit of 5000. This gas limit was hard coded into the protocol to ensure that miners and users could get up and running by installing clients during the initial launch of the protocol. The gas limit would later be increased to 3 million with the Frontier thawing fork (exactly 3,141,592). The canary contracts were contracts that gave a binary signal of either 0 or 1. These canary contracts were an initial launch mechanism used only during the Frontier release of Ethereum. If clients detected that multiple contracts had switched to a signal of 1, they would stop mining and urge the user to update their client. This prevented prolonged outages by ensuring that miners did not prevent a chain upgrade.
```
Comment (raxhvl) [inline docs/wiki/protocol/history.md:11]:
```suggestion
The Frontier release marked the launch of the Ethereum Protocol. It was launched on July 30, 2015, at 3:26:13 AM UTC, which is the timestamp of the [Ethereum genesis block](https://etherscan.io/block/0). Frontier launched with a gas limit of 5000. This gas limit was hard coded into the protocol to ensure that miners and users could get up and running by installing clients during the initial launch of the protocol. The gas limit would later be increased to 3 million with the Frontier thawing fork (exactly 3,141,592). The canary contracts were contracts that gave a binary signal of either 0 or 1. These canary contracts were an initial launch mechanism used only during the Frontier release of Ethereum. If clients detected that multiple contracts had switched to a signal of 1, they would stop mining and urge the user to update their client. This prevented prolonged outages by ensuring that miners did not prevent a chain upgrade.
```

File: docs/wiki/protocol/history.md (hunk 3 above)
Comment (raxhvl) [reply docs/wiki/protocol/history.md:22]: Could you add launch date and block number for homestead. Refer: https://consensys.io/blog/a-short-history-of-ethereum

**Review by raxhvl — CHANGES_REQUESTED** [review]
Thank you so much for contributing. I have left a few formatting nits and a suggestion. Rest looks good to me.

File: docs/wiki/protocol/history.md (hunk 3 above)
Comment (nokcha75) [reply docs/wiki/protocol/history.md]: I have pulled in the latest changes and merged the content. I also implemented your suggestions.

**Comment by nokcha75** [comment]
All suggestions are now implemented

**Comment by raxhvl** [comment]
Looks great!

**Review by raxhvl — APPROVED** [review]

**Pull request merged**